packages/docs/docs/app/bots.md

```diff
@@ -28,25 +25,120 @@ By using Bots, the entire logic is self contained and managed in one place. Like
 
 We have a full suite of detailed [Bot tutorials](https://docs.medplum.com/tutorials/bots/intro) that go through use cases in detail.
 
-## Create your first bot
+## Creating a Bot
+
+:::caution Note
+
+Bots are disabled by default. Contact your info@medplum.com if you'd like to learn more.
+
+:::
+
+:::caution Note
+
+Bots are restricted to Project administrators. If you do not have access, contact your Project administrator.
+
+:::
+
+To create a Bot, navigate to the [Project Admin panel](https://app.medplum.com/admin/project) and click "Create new Bot".
+
+![Create a Bot](/img/app/bots/create_bot.png)
 
-**Note:** Bots are disabled by default. Contact your info@medplum.com if you'd like to learn more.
+On the next page you can enter a bot **name** and **description** (optional). You can also optionally set an [**access
+policy**](app/access-control#access-policies) on the Bot, which can restrict the read/write privileges of the bot's
+code. By default, Bots have read/write access to all resources.
 
-**Note:** Bots are restricted to Project administrators. If you do not have access, contact your Project administrator.
+![Enter Bot Properties](/img/app/bots/enter_bot_properties.png)
 
-Create a Bot from the [Project Admin panel](https://app.medplum.com/admin/project).
+Click "Create Bot" to save the Bot, and you will see an acknowledgement that the Bot has been created.
 
-Enter a Bot name. For the first version of our Bot, let's enter the code:
+![Bot Created Acknowledgement](/img/app/bots/bot_created_acknowledgement.png)
+
+## Editing a Bot
+
+You can see all Bots in your account on the Bot resource page: [https://app.medplum.com/Bot](https://app.medplum.com/Bot)
+
+Click on your new Bot and navigate to the **Editor** tab. This presents a code editing window where you can write your
+Javascript code.
+
+![Bot Editor](/img/app/bots/BotResourcePage.gif)
+
+All Bots are simply a single Javascript file with a standard async entry point function called `handler`. To start your bot, create a `handler` function as shown below in the **Editor** window.
 
 ```javascript
```

Comment:
Maybe use typescript here?

```typescript
import { BotEvent, MedplumClient} from '@medplum/core';

export async function handler(medplum: MedplumClient, event: BotEvent): Promise<any> {
  // Your code here
}
```

Reply:
Trying to keep it in sync with the in-browser Editor window for right now which needs to be javascript (matches the rest of the tutorial)